monitor/docker.go

```diff
@@ -334,23 +324,15 @@ func (d *dockerMonitor) syncContainers() error {
 
 		if err != nil {
 			log.WithFields(log.Fields{
-				"package":   "monitor",
-				"monitor":   d,
-				"error":     err,
-				"container": container,
+				"package": "monitor",
```

Comment:
Yes, but the issue is that in some cases the "Container" object could be nil. (not given by the Docker API).